iroha-lib/examples/QueryExample.cpp

```diff
@@ -0,0 +1,147 @@
+            .createAsset(asset_name, domain_id, 0)
+            .signAndAddSignature();
+    return tx_proto;
+}
+
+
+void sendSampleTransaction(const std::string& account_name,
+                           const std::string& key_path,
+                           const std::string& peer_ip,
+                           uint16_t torii_port,
+                           uint32_t quorum,
+                           const std::string& domain_id,
+                           const std::string& user_default_role,
+                           const std::string& asset_name)
+{
+    const auto tx_proto = generateTransaction(account_name,
+                                              key_path,
+                                              domain_id,
+                                              user_default_role,
+                                              asset_name);
+
+    iroha_lib::GrpcClient(peer_ip, torii_port).send(tx_proto);
+
+    const auto tx_hash = getTransactionHash(tx_proto);
+    printTransactionStatus(peer_ip, torii_port, tx_hash);
+}
+
+
+void runQueryWithSingleTransactionGenerated(const char* key_path)
+{
+    auto account_name = "admin@test";
+    const auto peer_ip = "127.0.0.1";
+    uint16_t torii_port = 50051;
+    const auto quorum = 1;
+    const auto user_default_role = "user";
+
+    sendSampleTransaction(account_name,
+                          key_path,
+                          peer_ip,
+                          torii_port,
+                          quorum,
+                          "domainsamplev4",
+                          user_default_role,
+                          "assetnamesamplev4");
+
+    const auto query_proto = generateGetAccountAssetsQuery(account_name, key_path);
+    iroha_lib::GrpcClient(peer_ip, torii_port).send(query_proto);
+    assert(query_proto.payload().get_account_assets().account_id() == account_name);
+}
+
+
+void runQueryWithMultiplyTransactionsGenerated(const char* key_path)
+{
+    auto account_name = "admin@test";
+    const auto peer_ip = "127.0.0.1";
+    uint16_t torii_port = 50051;
+    const auto quorum = 1;
+    const auto user_default_role = "user";
+
+    uint8_t transactionCounter = 4u;
+    while (transactionCounter)
+    {
+        sendSampleTransaction(account_name,
+                              key_path,
+                              peer_ip,
+                              torii_port,
+                              quorum,
+                              "domainsamplequeryv" + std::to_string(transactionCounter),
+                              user_default_role,
+                              "assetnamesamplequeryv" + std::to_string(transactionCounter));
+        --transactionCounter;
+    }
+
+    const auto query_proto = generateGetAccountTransactionsQuery(account_name, key_path);
+    iroha_lib::GrpcClient(peer_ip, torii_port).send(query_proto);
+    assert(query_proto.payload().get_account_transactions().account_id() == account_name);
```

Comment:
I suggest to add sample of iteration over returned entries, like:
https://github.com/hyperledger/iroha-python/blob/0b260d480ac96ca1e33c0ea7e2a08a98e1378cc6/examples/tx-ordering.py#L149

Reply:
Refactored.